components/search/api-model/src/commonMain/kotlin/RunWithPackage.kt

```diff
@@ -31,5 +31,6 @@ data class RunWithPackage(
     val ortRunIndex: Long,
     val revision: String?,
     val createdAt: Instant,
-    val packageId: String,
+    val packageId: String?,
```

Comment:
Commit message nits:
- Double space in "pattern  used"
- "Note1" -> "Note 1" (same for 2)